enhancements/ota/distribute-secondary-metadata-as-container-image.md

```diff
@@ -0,0 +1,165 @@
+---
+title: cincinnati-graph-data-package-distribute-and-consume-in-a-container-image-repository
+authors:
+  - "@sjunker"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-05-07
+last-updated: 2020-05-07
+status: provision
+see-also:
+-
+replaces: []
+superseded-by: []
+---
+
+# cincinnati-graph-data: package, distribute, and consume in a container image repository
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+### Cincinnati Git(Hub) scraper deprecation
+We need to decide whether we want to maintain the Git(Hub) scraper plugin in the future or drop it altogether.
```

Comment:
+1 to drop it. Customers can mount the volume with a cloned repo / extracted tarball and configure operator to use it, in case they want to mutate the data

Reply:
> ...and configure operator to use it...

Not clear to me how you'd do that with the current operator config, but sure, extending the config is possible.  And yeah, with a Cincinnati operator to handle this sort of thing, I'd like to do it that way instead of having Git logic baked into Cincinnati.

Reply:
I think discussing the operator config exposure is out of scope for this enhancement.

Reply:
On the open question. I'll add the decision to deprecate the Git plugin to the proposal, mentioning that Cincinnati can be configured to read these files directly from a user-provided directory.